src/FusePoolDirectory.sol

```diff
@@ -234,7 +321,11 @@ contract FusePoolDirectory is OwnableUpgradeable {
      */
     function setPoolName(uint256 index, string calldata name) external {
         Comptroller _comptroller = Comptroller(pools[index].comptroller);
+
+        // Note: Compiler throws stack to deep if autoformatted with Prettier
+        // prettier-ignore
         require(msg.sender == _comptroller.admin() && _comptroller.adminHasRights() || msg.sender == owner());
```

Comment:
If auto-formatted by Prettier this becomes

```solidity
        require(
            (msg.sender == _comptroller.admin() &&
                _comptroller.adminHasRights()) || msg.sender == owner()
        );
```

Which should be semantically the same, just a more explicit way of indicating the order of execution.
The compiler on the other hand throws a `CompilerError: Stack too deep when compiling inline assembly: Variable dataEnd is 4 slot(s) too deep inside the stack.` if you do so.

Reply:
> If auto-formatted by Prettier this becomes
> 
> ```solidity
>         require(
>             (msg.sender == _comptroller.admin() &&
>                 _comptroller.adminHasRights()) || msg.sender == owner()
>         );
> ```
> 
> Which should be semantically the same, just a more explicit way of indicating the order of execution. The compiler on the other hand throws a `CompilerError: Stack too deep when compiling inline assembly: Variable dataEnd is 4 slot(s) too deep inside the stack.` if you do so.

not sure the root cause yet, but this compiles successfully with optimizer set to false. Let's just keep the ignore for now